cartridges/rvw_dev_console/cartridge/client/default/scss/_toast.scss

```diff
@@ -71,7 +71,7 @@
 }
 
 .v-notices .v-toast-warning {
-    background-color: #ffc107;
+    background-color: #f38518;
```

Comment:
@z1haze this was a contrast ratio issue that I fixed with background color not being dark enough.